domainlab/models/a_model.py

```diff
@@ -33,7 +33,7 @@ def cal_loss(self, tensor_x, tensor_y, tensor_d=None, others=None):
         """
         list_loss, list_multiplier = self.cal_reg_loss(tensor_x, tensor_y, tensor_d, others)
         loss_reg = self.inner_product(list_loss, list_multiplier)
-        return self.multiplier4task_loss * self.cal_task_loss(tensor_x, tensor_y) + loss_reg
+        return self.multiplier4task_loss() * self.cal_task_loss(tensor_x, tensor_y) + loss_reg
```

Comment:
In the latest version, I have defined this as a property, so no need to have () here?

Reply:
@e-dorigatti , shall I or you close this PR?

Reply:
probably this issue occurred on the cluster because I did not pull the latest version. I will close it later once I make sure it works (it probably does)